code/ATMOSPHERICS/components/binary_devices/pipeturbine.dm

```diff
@@ -73,15 +73,15 @@
 			network2.update = 1
 
 	update_icon()
-		cut_overlays()
+		overlays.Cut()
```

Comment:
every single one of these is going to brake the icons

Reply:
changed as many as I've found